### PR TITLE
Add let and const

### PR DESCRIPTION
**Pull request by JulianPasquale**

### CODE REVIEW

File: src/slides/class1.jsx [hunk 1]
```diff
@@ -124,5 +137,286 @@ export default () => (
...
+      </CodePane>
+    </Slide>
+
+    <Slide backgroundColor="#ffd84d">
+      <Heading margin="0px" fontSize="100px">
+        Variables
+      </Heading>
+
+      <Text>
+        JavaScript es un lenguaje dinámico y de tipado flexible. En parte esto significa que las variables no están asociadas directamente con ningún tipo de valor en particular y se les pueden asignar o reasignar valores de todos los tipos.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:170]:
```suggestion
        ... valores de cualquier tipo.
```

File: src/slides/class1.jsx [hunk 2]
```diff
@@ -124,5 +137,286 @@ export default () => (
...
+      </CodePane>
+    </Slide>
+
+    <Slide backgroundColor="#ffd84d">
+      <Heading margin="0px" fontSize="50px">
+        Constantes
+      </Heading>
+
+      <Text>
+        A diferencia de las variables, las constantes no pueden cambiar su valor posteriorment. Se debe especificar el valor al momento de su declaración.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:191]: Typo in posteriorment**e**

File: src/slides/class1.jsx [hunk 3]
```diff
@@ -124,5 +137,286 @@ export default () => (
...
+
+      <CodePane language="js">
+        {`
+          const FOO = "bar";
+
+          // this will throw an error - Uncaught TypeError: Assignment to constant variable.
+          FOO = 20;
+
+          // trying to redeclare a constant throws an error
+          // Uncaught SyntaxError: Identifier 'MY_FAV' has already been declared
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:206]: Identifier should be 'FOO'

File: src/slides/class1.jsx [hunk 4]
```diff
@@ -124,5 +137,286 @@ export default () => (
...
+          const FOO = 20;
+        `}
+      </CodePane>
+
+      <Notes>
+        Constants can be declared with uppercase or lowercase, but a common convention is to use all-uppercase letters.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:212]: Should this be in spanish? I'm not sure about the part of `a common convention is to use all-uppercase letters.`. I mean, we usually declare a lot of `consts` and almost none of them are all-uppercase.

File: src/slides/class1.jsx [hunk 5]
```diff
@@ -124,5 +137,286 @@ export default () => (
...
+                `}
+            </CodePane>
+
+            <iframe src="/src/slides/class1/examples/boolean4.html"></iframe>
+          </FlexBox>
+        </Grid>
+      </div>
+
+      <Notes>
+        No confundir el objeto Booelan con los valores primitivos Boolean ni con la función Boolean.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:390]: Typo here in the first Boo**e**lan

Comment (nachosoloaga) [reply src/slides/class1.jsx:390]: Also, we should see how we explain this in order to not confuse people 😄

File: src/slides/class1.jsx [hunk 6]
```diff
@@ -124,5 +137,286 @@ export default () => (
...
+
+    <Slide backgroundColor="#ffd84d">
+      <div style={{width: '80%'}}>
+
+        <Heading margin="0px" fontSize="50px">
+          Null
+        </Heading>
+
+        <Text>
+          Este valor se utiliza para representar la no existencia o invalidez de algo, generalmente de manera intencional.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:404]: We could add something about Null vs undefined? Maybe not here but at the time speaking

File: src/slides/class1/examples/boolean1.html [hunk 7]
```diff
@@ -0,0 +1,26 @@
+<html lang="en">
```
Comment (nachosoloaga) [inline src/slides/class1/examples/boolean1.html:1]: These examples are full facha

**Review by nachosoloaga — APPROVED** [review]

File: src/slides/class1.jsx (hunk 4 above)
Comment (JulianPasquale) [reply src/slides/class1.jsx:210]: copy-paste from here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage.

This is a Note, so it's not part of the slide - only we'll be able to see this in presenter mode. But agree, we should do all the notes in Spanish

**Pull request merged**